lib/hasPermissions.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
please avoid such thing. It's a bad thing to disable eslint for no strong reason. let's not disable this rule.